iq-config-as-code/iq-scrape-conf.py

```diff
@@ -104,20 +104,22 @@ def main():
             org_conf = {}
             org_conf = org_configuration(org)
             org_apps = []
-            for app in applications:
-                if app['organizationId'] == org['id']:
-                    if in_scope(app=app):
-                        org_apps.append(app_configuration(app))
-            if len(org_apps) or in_scope(org=org):
-                org_conf['applications'] = org_apps
-                od = {}
-                od['organizations'] = []
-                od['organizations'].append(org_conf)
-                if (org_conf['name'] == 'Root Organization'):
-                    orgs.insert(0, org_conf)
-                else:
-                    orgs.append(org_conf)
-                persist_data(od, f'{output_dir}{get_organization_name(org["id"])}-config.json')
+            if applications is not None:
+                app =[]
```

Comment:
Probably this is not needed, is it?

Reply:
d60935ddde7464ea5c4098e6dcfdb7f384827263